src/__tests__/jsonApiLink.ts

```diff
@@ -2036,116 +2054,124 @@ describe('Mutation', () => {
         );
       }
     });
-  });
 
-  it.skip('returns an error on zero Content-Length but status > 300', async () => {
-    expect.assertions(1);
+    it('returns an error on zero Content-Length but status > 300', async () => {
+      expect.assertions(1);
 
-    const link = new JsonApiLink({ uri: '/api' });
+      const link = new JsonApiLink({ uri: '/api' });
 
-    const post = { id: '1', title: 'Love apollo' };
-    fetchMock.post('/api/posts', {
-      headers: { 'Content-Length': 0 },
-      status: 500,
-      body: post,
-    });
+      const post = { id: '1', title: 'Love apollo' };
+      fetchMock.post('/api/posts', {
+        headers: { 'Content-Length': 0 },
+        status: 500,
+        body: post,
+      });
 
-    const createPostMutation = gql`
-      fragment PublishablePostInput on REST {
-        title: String
-      }
+      const createPostMutation = gql`
+        fragment PublishablePostInput on REST {
+          title: String
+        }
 
-      mutation publishPost($input: PublishablePostInput!) {
-        publishedPost(input: $input)
-          @jsonapi(type: "Post", path: "/posts", method: "POST") {
-          id
-          title
+        mutation publishPost($input: PublishablePostInput!) {
+          publishedPost(input: $input)
+            @jsonapi(path: "/posts", method: "POST") {
+            id
+            title
+          }
         }
-      }
-    `;
-    return await makePromise<Result>(
-      execute(link, {
-        operationName: 'publishPost',
-        query: createPostMutation,
-        variables: { input: { title: post.title } },
-      }),
-    ).catch(e =>
-      expect(e).toEqual(
-        new Error('Response not successful: Received status code 500'),
-      ),
-    );
+      `;
+      return await makePromise<Result>(
+        execute(link, {
+          operationName: 'publishPost',
+          query: createPostMutation,
+          variables: { input: { title: post.title } },
```

Comment:
This input isn't JSON API format, but I guess it's not really relevant for this test.

Reply:
Good catch. I'll go ahead and update it, just so it's all consistent and doesn't throw anyone off if they are looking at the tests for examples.